packages/driver/cypress/integration/e2e/multidomain_spec.ts

```diff
@@ -48,21 +48,25 @@ describe('multidomain', { experimentalSessionSupport: true }, () => {
       expectTextMessage('form:submitted', done)
 
       // @ts-ignore
-      cy.switchToDomain('127.0.0.1:3501', () => {
-        Cypress.on('form:submitted', () => {
+      cy.switchToDomain('foobar.com', () => {
+        Cypress.once('form:submitted', () => {
           top!.postMessage({ host: location.host, actual: 'form:submitted' }, '*')
         })
 
         cy.get('form').submit()
       })
     })
 
-    it('window:before:unload', (done) => {
+    // FIXME: reloading the page is problematic because the proxy delays the
+    // request, but the driver currently waits for a switchToDomain, which
+    // has already been called and won't be called again. need to handle any
+    // sort of page reloading in the AUT when it's cross-domain
+    it.skip('window:before:unload', (done) => {
```

Comment:
Is there an issue / task we can link to make sure this isnt' forgotten?

Reply:
This is covered by a story in the Miro board for multi-domain. It will be written up as a github issue eventually. I'm not too concerned about this being forgotten.